### PR TITLE
Fix/puppet/missing semantic puppet

### PR DESCRIPTION
**Pull request by the-eater**
Currently if puppet is installed, it will complain about missing semantic_puppet

### CODE REVIEW

File: srcpkgs/puppet/template [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 pkgname=puppet
 version=6.1.0
-revision=1
+revision=2
 noarch=yes
 build_style=ruby-module
 hostmakedepends="ruby facter-devel hiera"
 makedepends="facter-devel"
-depends="ruby>=2.2 facter hiera"
+depends="ruby>=2.2 facter hiera ruby-semantic_puppet"
```
Comment (maxice8) [inline srcpkgs/puppet/template]: I think the ruby>=2.2 can have the version specifier removed, we are on 2.6.0 now

Comment (the-eater) [reply srcpkgs/puppet/template]: Do you want that in a seperate commit?

**Comment by maxice8** [comment]
On Thu, Jan 24, 2019 at 03:16:31PM -0800, eater wrote:
> the-eater commented on this pull request.
> 
> 
> 
> >  noarch=yes
>  build_style=ruby-module
>  hostmakedepends="ruby facter-devel hiera"
>  makedepends="facter-devel"
> -depends="ruby>=2.2 facter hiera"
> +depends="ruby>=2.2 facter hiera ruby-semantic_puppet"
> 
> Do you want that in a seperate commit?
> 
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/7688#discussion_r250814445

Same commit

**Comment by the-eater** [comment]
Done

**Pull request merged**